Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:2.7
+
+MAINTAINER Lucas Manuel Rodriguez <lucas@spideroak-inc.com>
```

Comment:
It seems that `MAINTAINER` is deprecated, you should go with `LABEL`, see https://docs.docker.com/engine/reference/builder/#maintainer-deprecated